src/main/drivers/pwm_output_dshot_hal.c

```diff
@@ -122,13 +128,11 @@ void pwmDshotSetDirectionOutput(
 }
 
 
-FAST_CODE void pwmCompleteDshotMotorUpdate(uint8_t motorCount)
+FAST_CODE void pwmCompleteDshotMotorUpdate(void)
 {
-    UNUSED(motorCount);
-
     /* If there is a dshot command loaded up, time it correctly with motor update*/
-    if (pwmDshotCommandIsQueued()) {
-        if (!pwmDshotCommandOutputIsEnabled(motorCount)) {
+    if (dshotCommandIsQueued()) {
+        if (!dshotCommandOutputIsEnabled(dshotPwmDevice.count)) {
```

Comment:
`&&`

Reply:
Yes.

There's another instance in `pwm_output_dshot_hal_hal.c`; I'll apply the same change.